routes/auth/user/getUsers.ts

```diff
@@ -2,15 +2,15 @@ import { db, event } from "#src/utils";
 
 export const getUsers = async () => {
   const limit = event.queryStringParameters?.limit ?? 10;
-  const offset = event.queryStringParameters?.page ?? 0;
+  const offset = event.queryStringParameters?.offset ?? 0;
   const sort = event.queryStringParameters?.sort === "desc" ? "DESC" : "ASC";
   const sortBy = event.queryStringParameters?.sortBy ?? "id";
 
   await db.connect();
   const rows = (
     await db.query({
-      text: `SELECT * FROM "users" ORDER BY ${sortBy} ${sort} LIMIT $1 OFFSET $2`,
-      values: [limit, offset],
+      text: `SELECT * FROM "users" ORDER BY $3 $4 LIMIT $1 OFFSET $2`,
```

Comment:
Swapped over to this rather than direct JS interpolation, not sure if that was safe against injection. This is probably the way to go.